libs/akita/src/__tests__/queryEntity.spec.ts

```diff
@@ -1098,13 +1099,46 @@ describe('selectAll - limit to and filterBy and sorting', () => {
     let res;
     subscription = query
       .selectAll({
-        filterBy: el => el.value === 5,
+        filterBy: (el) => el.value === 5,
         limitTo: 8,
-        sortBy: 'id'
+        sortBy: 'id',
       })
-      .subscribe(_res => {
+      .subscribe((_res) => {
         res = _res;
       });
     expect(res.length).toBe(7);
   });
 });
+
+describe('track entity ids', () => {
+  const store = new TodosStore();
+  const query = new QueryEntity(store);
+
+  it('should track new entity id', () => {
+    let res;
+
+    store.add({ id: 1, title: 'a' });
+
+    query
+      .selectEntity(1)
+      .pipe(trackIdChanges(store))
+      .subscribe((_res) => (res = _res));
+
+    expect(res.id).toBe(1);
+    expect(res.title).toBe('a');
+    expect(Object.keys(store._value().entities).length).toBe(1);
+    expect(Object.keys(store._value().ids).length).toBe(1);
+
+    store.update(1, { id: 2, title: 'a' });
+    expect(res.id).toBe(2);
+    expect(res.title).toBe('a');
+    expect(Object.keys(store._value().entities).length).toBe(1);
+    expect(Object.keys(store._value().ids).length).toBe(1);
+
+    store.update(2, { id: 3, title: 'a' });
```

Comment:
Please, add one section that updates one of the current entity props ( not the `id`)